addons/website/models/website.py

```diff
@@ -186,7 +186,7 @@ def _compute_menu(self):
             website.menu_id = top_menus and top_menus[0].id or False
 
     # self.env.uid for ir.rule groups on menu
-    @tools.ormcache('self.env.uid', 'self.id')
+    @tools.ormcache('self.env.uid', 'self.id', cache='templates')
```

Comment:
Actually I did it because a tour `test_15_website_link_tools` was breaking without that and I didn't got the "detailed why". 

I think that some invalidation is missing somewhere and the fact that both are in the same cache fixes the issue.
I would like to give you the exact why but maybe later because I already took a few hours to investigate without success and I'm reaching my limits on this specific one.
This makes sense (kind of) to keep it together anyway.

About the naming, I'm open to suggestions :)